rulesets/data/es_lookup_by_cz.csv

```diff
@@ -0,0 +1,276 @@
+type,subtype,program,1A,1B,1C,2A,2B,2C,3A,3B,3C,4A,4B,4C,5A,5B,5C,6A,6B,6C,7,8
```

Comment:
Let's remove the program column and use a separate file for each program. (The file can also include just the climate zone columns that apply.)